src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -3,11 +3,13 @@
 import java.util.Random;
 
 public class FigureSupplier {
+    private static final int RANDOM_NUMBER_BOUND = 100;
```

Comment:
Are you sure that you use it?
You create a constant with value 100, but when you try to get random number, you use Shape.values().length values. 
Just use value of your RANDOM_NUMBER_BOUND constant instead of length